### PR TITLE
New form + routing + bug fixes

### PR DESCRIPTION
**Pull request by petrvecera**

### CODE REVIEW

File: App/app/controller/StoreLoadController.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ Ext.define('Enif.controller.StoreLoadController', {
         // clear the filters on the store before loading
         gamesStore.clearFilter();
```
Comment (HonzaTuron) [inline App/app/controller/StoreLoadController.js:24]: Is double clearFilter call necessary?

Comment (petrvecera) [reply App/app/controller/StoreLoadController.js:24]: I know about that and I don't know, the comment says clear the filter before loading, I didn't wanted to investigate further

**Comment by petrvecera** [comment]
By the power given by god I shall merge this pull request without any approved review.

**Comment by ghost** [comment]
:-D

**Pull request merged**